radixdlt-core/radixdlt/src/main/java/com/radixdlt/store/tree/TreeUtils.java

```diff
@@ -34,6 +35,33 @@ public static String toHexString(byte[] byteBuffer) {
 			return Arrays.concatenate(new byte[] {(byte) oddPrefix}, rawKey);
 		}
 	}
+
+	/**
+	 * Convert an array of nibbles to a byte array. Starting from the beginning of the array, it converts each pair
+	 * of nibbles into a byte. The first nibble is the 4 most significant bits and the second one the 4 least.
+	 * <br/>
+	 * Examples: <br/>
+	 *	[0, 1, 0, 2] becomes [1, 2] <br/>
+	 *  [1, 1, 2, 2] becoenes [17, 34] <br/>
+	 *
+	 * @param nibbles nibbles array to be converted
```

Comment:
repetition

Reply:
not necessarily a repetition as the first one is the name of the parameter and second is a description, but I removed it anyway